modules/fuse-box-responsive-api/index.js

```diff
@@ -9,10 +9,11 @@
 
     /* @if universal */
     var isBrowser = typeof window !== "undefined";
+    /* @if !npm */
     if (!isBrowser) {
```

Comment:
yes, so if target is `npm`, there should not be any `global.require = require` - that is what I removed here, saying that is line is used only when the target is not npm.